src/StyledComponents.js

```diff
@@ -483,3 +484,108 @@ export const WeOfferHeader = styled.h3`
 export const WeOfferUL = styled.ul`
   color: ${secondary};
 `;
+
+//Code of Conduct Modal *********************************************************
```

Comment:
This file is getting quite large, how many more components we going to keep putting in here? We should start thinking of how to break this apart into smaller files that are easier to manage.

Reply:
I created a style file for Careers and CodeOfConduct folder, I will start doing this from now on.